docs/source/daml/reference/interfaces.rst

```diff
@@ -10,10 +10,7 @@ Reference: Interfaces
   This feature is under active development and not officially supported in
   production environments.
 
-In Daml, an interface defines an abstract type which specifies the behavior
-that a template must implement. This allows decoupling such behavior from its
-implementation, so other developers can write applications in terms of the
-interface instead of the concrete template.
+In Daml, an interface defines an abstract type together with a behavior specified by its view type, method signatures, and choices. For a template to be considered an instance of this interface, there must be a corresponding ``interface instance`` clause where all the methods of the interface (including the special ``view`` method) are implemented. This allows decoupling such behavior from its implementation, so other developers can write applications in terms of the interface instead of the concrete template.
```

Comment:
In general, I would recommend preserving the existing line breaks to make reviewing easier.

Reply:
fair point, I'll reflow these in a new commit